tests/Alcaeus/MongoDbAdapter/Mongo/MongoCollectionTest.php

```diff
@@ -1157,11 +1158,17 @@ public function testGetIndexInfo()
                 'name' => 'foo_1',
                 'ns' => 'mongo-php-adapter.test',
             ],
+            [
+                'v' => 1,
+                'key' => ['bar' => 1],
+                'name' => 'bar_1',
+                'ns' => 'mongo-php-adapter.test',
+                'unique' => true,
+            ],
         ];
 
-        $this->assertSame(
-            $expected,
-            $collection->getIndexInfo()
+        $this->assertTrue(
```

Comment:
You can use `assertEquals` instead of `assertSame` which does the same thing here but is more explanatory.